src/EnsembleKalmanInversions.jl

```diff
@@ -111,8 +114,14 @@ Keyword Arguments
 
 - `forward_map_output`: Default: `nothing`.
 
+- `process`: The Ensemble Kalman process. Default: `Inversion().
+
+- `tikhonov`: Whether to incorporate prior information in the EKI objective via Tikhonov regularization.
+    See Chada et al. "Tikhonov Regularization Within Ensemble Kalman Inversion." SIAM J. Numer. Anal. 2020.
```

Comment:
Can you clarify when this is used? Does this pertain to adaptive stepping (eg to a particular scheme?) or does this pertain to information stored in the iteration summaries?